### DISCUSSION TITLE
Is it possible to automatically remove intermediate Docker images?

### DISCUSSION THREAD

**Discussion by carry0987** [Noelware/docker-manifest-action#407] [Ideas]
As the title suggests, when we configure an action as shown below:
```
inputs: namespace/image:latest
images: namespace/image:latest-amd64,namespace/image:latest-arm64
```

It successfully creates the `namespace/image` with `amd64` and `arm64` support. However, it also leaves the separate images `namespace/image:latest-amd64` and `namespace/image:latest-arm64` intact. Is there a way to have these intermediate images removed automatically?

**Comment by auguwu** [comment]
Not really since it needs those both so that container implementations can use x or y depending on the architecture.

If you’re wanting to have one image to support both, this action isn’t for you since you can just use the —-platforms argument in docker buildx build.
